QRCoder/Builders/Payloads/PayloadBase.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace QRCoder.Builders.Payloads
+{
+    public abstract class PayloadBase : IPayload, IConfigurableEccLevel, IConfigurableEciMode, IConfigurableVersion
```

Comment:
The idea here is that while most all payloads can inherit from `PayloadBase` and inherit some configurable settings, perhaps some payloads, like the swiss banking payload, does not support configuring the Eci mode or whatever.  Or, users can override these properties to add validation, ensuring that a certain minimum ECC level is maintained.  There's a number of ways to actually implement this, all with similar end results.